pkgs/development/libraries/eccodes/default.nix

```diff
@@ -23,6 +23,7 @@ stdenv.mkDerivation rec {
                   openjpeg
                   libpng
                   gfortran
+                  perl
```

Comment:
Is it possible this is just used at build-time and should therefore be in `nativeBuildInputs`?